pi5/board.go

```diff
@@ -285,80 +235,6 @@ func findNewDigIntConfig(
 	return nil
 }
 
-func (b *pinctrlpi5) reconfigureInterrupts(newConf *LinuxBoardConfig) error {
```

Comment:
...don't we still need this? We still have interrupts on the board, right?

Reply:
Yes we still have interrupts but they are no longer in the config. You can still create one by calling `DigitalInterruptByName(pinnum)`,they don't need reconfiguration anymore.

Reply:
That sounds like the interrupts must be named identically to the pins they're on, and you can't give them different names (e.g., "left_encoder" instead of "40"). This will break certain configs, but perhaps we're still early enough in the company's history that only employees are using that feature and we can remove it anyway.

Reply:
yes thats correct, it would break configs in the regular pi5 driver but since this is a brand new module there are no configs to break. but yes something to consider if we ever switch to this module being the only pi5 implementation.